staging/src/k8s.io/apiserver/pkg/storage/tests/cacher_test.go

```diff
@@ -466,12 +477,32 @@ func TestWatch(t *testing.T) {
 	_ = updatePod(t, etcdStorage, podFooBis, fooUpdated)
 
 	verifyWatchEvent(t, nowWatcher, watch.Modified, podFooBis)
+
+	// Make current watch cache full, 100 is watch cache initial capacity.
+	// Make start and last event duration exceed eventFreshDuration(current 75s) to ensure watch cache won't expand.
+	for i := 0; i < 100; i++ {
+		fakeClock.SetTime(time.Now().Add(time.Duration(i) * time.Minute))
+		podFoo := makeTestPod(fmt.Sprintf("foo-%d", i))
+		podFoo.Spec.NodeName = fmt.Sprintf("fakeNode-%d", i)
```

Comment:
Is this really needed for anythng?

Reply:
Not needed but a copy-paste of formers. I'll drop this line later.